backend/entities/academics/section_entity.py

```diff
@@ -79,6 +74,12 @@ class SectionEntity(EntityBase):
         primaryjoin="and_(SectionEntity.id==SectionMemberEntity.section_id, SectionMemberEntity.member_role!='STUDENT')",
     )
 
+    # All applicants where section is preferred
+    # NOTE: This field establishes a many-to-many relationship between the sections and applications table.
+    preferred_applicants: Mapped[list["UTAEntity"]] = relationship(
+        secondary=section_application_table, back_populates="preferred_sections"
```

Comment:
This relationship might lead to an issue over time and as more terms pass, since this list would not contain only applicants in the most recent application cycles. I recommend taking a look at the `primaryjoin=` examples used in the relationship fields defined above to filter this to only include future terms.